roles/keycloak_quarkus/tasks/config_store.yml

```diff
@@ -0,0 +1,64 @@
+---
+- name: "Check if keytool exists in path"
+  block:
```

Comment:
This block happen quite late in the deployment; would you mind moving it to the prereq tasks? change_when below can safely be set to false

Reply:
The check now happens after having installed JVM in `fastpackages.yml`, so keytool should definitely be there, but  a check never hurts either ;)